sklearn/neighbors/_regression.py

```diff
@@ -112,23 +112,12 @@ class KNeighborsRegressor(KNeighborsMixin, RegressorMixin, NeighborsBase):
     n_samples_fit_ : int
         Number of samples in the fitted data.
 
-    Examples
-    --------
-    >>> X = [[0], [1], [2], [3]]
-    >>> y = [0, 0, 1, 1]
-    >>> from sklearn.neighbors import KNeighborsRegressor
-    >>> neigh = KNeighborsRegressor(n_neighbors=2)
-    >>> neigh.fit(X, y)
-    KNeighborsRegressor(...)
-    >>> print(neigh.predict([[1.5]]))
-    [0.5]
-
     See Also
     --------
-    NearestNeighbors
-    RadiusNeighborsRegressor
-    KNeighborsClassifier
-    RadiusNeighborsClassifier
+    NearestNeighbors : TEST1.
+    KNeighborsRegressor : TEST2.
+    KNeighborsClassifier : TEST3.
+    RadiusNeighborsClassifier : TEST4.
```

Comment:
I am guessing this is why this PR is in still in Draft?

Reply:
yes!

Reply:
Hey Thomas, I added this missing descriptions but I'm having problems to push the changes because the remote branch is newer than my local branch. Tried to merge the remote first but had no success!

Reply:
You can try to fix the merge conflicts by running:

```python
git fetch upstream main
git merge upstream/main
```

> Tried to merge the remote first but had no success!

What kind of trouble are you having?

Reply:
Hi Thomas! Sorry for my late answer. I think I solved the issues I was having by coming back to them with a fresh mind. Already made the PR "ready for review". I'll be paying attention to this chat so that I can finish the PR before the Post Sprint Event on Saturday.